EvoS.DirectoryServer/Inventory/InventoryManager.cs

```diff
@@ -59,5 +56,11 @@ public static InventoryComponent GetInventoryComponent(long accountId)
             return new InventoryComponent();
         }
 
+        public static Boolean BannerIsForeground(int bannerID) 
+        {
+            List<int> fg = new List<int>{ 200, 201, 202, 203, 204, 205, 206, 207, 208, 209, 210, 211, 212, 213, 214, 215, 251, 252, 253, 239, 240, 241, 245, 246, 247, 65, 242, 243, 244, 236, 237, 238, 248, 249, 250, 222, 299, 300, 301, 302, 303, 304, 305, 306, 307, 308, 309, 310, 311, 312, 313, 314, 315, 316, 317, 318, 319, 320, 321, 322, 323, 336, 337, 338, 339, 340, 341, 342, 343, 345, 347, 350, 351, 352, 353, 354, 355, 333, 334, 335, 361, 362, 363, 364, 365, 366, 367, 368, 369, 370, 358, 359, 360, 357, 376, 377, 378, 286, 390, 388, 391, 387, 386, 389, 392, 393, 398, 399, 400, 401, 402, 403, 404, 405, 406, 407, 408, 409, 410, 411, 412, 423, 424, 217, 434, 436, 435, 438, 437, 439, 440, 441, 444, 446, 442, 443, 456, 457, 458, 459, 460, 38, 465, 466, 324, 325, 326, 327, 328, 356, 467, 468, 475, 476, 477, 478, 479, 480, 481, 482 };
```

Comment:
It could be a set, and all collections in this class could be static but we can fix it later